File: src/BukkitPE/event/entity/ProjectileHitEvent.php
<?php
namespace BukkitPE\event\entity;

use BukkitPE\entity\Projectile;

class ProjectileHitEvent extends EntityEvent{
	public static $handlerList = null;

	/**
	 * @param Projectile $entity
	 */
	public function __construct(Projectile $entity){
		$this->entity = $entity;

	}

	/**
	 * @return Projectile
	 */
	public function getEntity(){
		return $this->entity;
	}

}

File: src/main/java/net/BukkitPE/event/entity/ProjectileHitEvent.java
package net.BukkitPE.event.entity;

import net.BukkitPE.entity.Entity;
import net.BukkitPE.entity.projectile.EntityProjectile;
import net.BukkitPE.event.HandlerList;

/**
 * BukkitPE Project
 */
public class ProjectileHitEvent extends EntityEvent {
    private static final HandlerList handlers = new HandlerList();

    public ProjectileHitEvent(EntityProjectile entity) {
        this.entity = entity;
    }

    public static HandlerList getHandlers() {
        return handlers;
    }

    @Override
    public Entity getEntity() {
        return super.getEntity();
    }
}
